regra_de_negocio/global_settings.py

```diff
@@ -17,3 +15,7 @@ def edit_global_settings(quantidade_sprint, dias_sprint):
     dados_global_settings = open("dados/global_setting.json", "w")
     json.dump(settings_atual, dados_global_settings)
     dados_global_settings.close()
+
+def obter_quantidade_sprint():
+    gs = read_global_settings()
```

Comment:
Essa global_settings deve sofrer uma alteraçãoo no banco. Pq nosso banco hoje precisa ter globalsettings = x e turma = x.
e ai esse read precisa buscar a globalsettings com a chave da turma. E ai o obter_quantidade_sprint vai voltar a globalsettings correta, e caso ela não exista, if gs is none, vc setta aqui um valor default 4.